.github/workflows/test.yml

```diff
@@ -5,6 +5,8 @@ concurrency:
 on:
   pull_request: { types: [opened, reopened, synchronize, ready_for_review] }
   push: { branches: [ main ] }
+env:
+  LOG_LEVEL: debug
```

Comment:
Is this supposed to be here or is it an artifact of debugging

Reply:
Truth be told, I haven't really decided. I usually leave logging at `info` in CI to prevent excessively long job output, but I've been bitten by that choice a few times.